concourse/client.py

```diff
@@ -249,6 +252,13 @@ def pipeline_cfg(self, pipeline_name: str):
         not_empty(response)
         return PipelineConfig(response, concourse_api=self, name=pipeline_name)
 
+    def pipeline_resources(self, pipeline_names):
+        if not isinstance(pipeline_names, list):
+            pipeline_names = [pipeline_names]
+
+        resources = map(lambda name: self.pipeline_cfg(pipeline_name=name).resources, pipeline_names)
+        yield from reduce((lambda x,y: x+list(y)), resources, [])
```

Comment:
use a for-loop instead, please. this should improve readability. still use yield from in the loop's body